dockstore-webservice/src/main/resources/migrations.1.6.0.xml

```diff
@@ -348,6 +348,23 @@
             ALTER TABLE sourcefile ADD CONSTRAINT absolutepath_check CHECK (absolutepath LIKE '/%' OR absolutepath='')
         </sql>
     </changeSet>
+
+    <changeSet author="louisecabansay (generated)" id="addStarredOrganizations">
+        <createTable tableName="starred_organizations">
+            <column name="organizationid" type="BIGINT">
+                <constraints primaryKey="true" primaryKeyName="starred_organizations_pkey"/>
+            </column>
+            <column name="userid" type="BIGINT">
+                <constraints primaryKey="true" primaryKeyName="starred_organizations_pkey"/>
+            </column>
+        </createTable>
+    </changeSet>
+    <changeSet author="louisecabansay (generated)" id="addUserIdStarOrgConstratint">
+        <addForeignKeyConstraint baseColumnNames="userid" baseTableName="starred_organizations" constraintName="fk_userIdStarredOrganizations" referencedColumnNames="id" referencedTableName="enduser"/>
+    </changeSet>
+    <changeSet author="louisecabansay (generated)" id="addOrganizationIdStarOrgConstratint">
+        <addForeignKeyConstraint baseColumnNames="organizationid" baseTableName="starred_organizations" constraintName="fk_organizationIdStarredOrganizations" referencedColumnNames="id" referencedTableName="organization"/>
+    </changeSet>
```

Comment:
Any time I add migrations to the migrations file, I try to add to the end of the file. Sometimes due to merging with develop to keep up to date my new migration will be moved from the bottom, but I move it back to the bottom. This is because the order of the migrations is sometimes important, so best to be safe.